tests/test-components/GitPanel.spec.tsx

```diff
@@ -78,10 +78,31 @@ describe('GitPanel', () => {
     await props.model.ready;
   });
 
+  describe('#constructor()', () => {
+    it('should return a new instance', () => {
+      const panel = new GitPanel(props);
+      expect(panel).toBeInstanceOf(GitPanel);
+    });
+
+    it('should set the default commit message summary to an empty string', () => {
+      const panel = new GitPanel(props);
+      expect(panel.state.summary).toEqual('');
+    });
+
+    it('should set the default commit message description to an empty string', () => {
+      const panel = new GitPanel(props);
+      expect(panel.state.summary).toEqual('');
```

Comment:
I presume the intention is:

```suggestion
      expect(panel.state.description).toEqual('');
```